### ISSUE TITLE
Hey Hey

### ISSUE THREAD

**Issue by grmkris** [daoleno/lenskit#1]
Hey I just noticed your library.. Really cool !
I wonder do I get typescript autocompletion for the queries?

**Comment by daoleno** [comment]
Yes, but it's still a work in progress. I will be wrapping some common queries into hooks with type definitions. Once that is done I will update it here.
